doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -48,22 +51,109 @@ When you ``cd`` into the new directory, you will see a folder structure as shown
      ├── proj-one
      │   ├── __init__.py
      │   └── proj_one_code.py
-     ├── proj-two
-     │   ├── __init__.py
-     │   └── proj_two_code.py
      └── tests
           ├── __init__.py
           └── test_shared_functions.py
 
-Please take a look at those files created with your favorite IDE software (e.g., Visual Studio Code, PyCharm, etc.).
+If you would like to add more projects, you can create a new folder under the root directory. For example, you can create a folder called ``proj-two`` and add your code there. This can be extrapolated for as many projects as you like:
 
-See the descriptions below for each file created in the project:
+.. code-block:: bash
+
+     mkdir proj-two
+     cd proj-two
+     touch __init__.py
+     touch proj_two_code.py
+
+
+File descriptions
+^^^^^^^^^^^^^^^^^^
+
+Now that the folder structure is created, let's go through the files and folders in the project:
+
+- ``README.md`` is a markdown file for project documentation. Typically, it includes a project description, installation instructions, and usage examples. You can edit this file to add your own project information.
+
+
+- ``shared_functions.py`` is where you define reused code. It includes the following example function to help demonstrate the concept of shared functions. In your project, this is where you would add the code you want to reuse:
```

Comment:
mention that this is an example file?  It can have any name and there can be any number of these files, but they need to be in the same directory.  \skpkg adds this file to get the user started, but just add more files as needed.